app/src/main/java/src/Chessboard.java

```diff
@@ -1,25 +1,128 @@
 package src;
 
+import java.time.Period;
+import java.util.ArrayList;
+import java.util.List;
+
 public class Chessboard {
-    public final int ASCI_CODE_A = 97;
+    public final int ASCI_CODE_A = 104;
     private final int DIMENSION = 8;
-    private String[][] board = new String[DIMENSION][DIMENSION];
+    public static Piece[][] board;
+    private List<Piece> pieces;
     boolean turn;
+    private Piece WRook1;
+    private Piece WHorse1;
+    private Piece WBishop1;
+    private Piece WQueen;
+    private Piece WKing;
+    private Piece WBishop2;
+    private Piece WHorse2;
+    private Piece WRook2;
+    private Piece WPeon1;
+    private Piece WPeon2;
+    private Piece WPeon3;
+    private Piece WPeon4;
+    private Piece WPeon5;
+    private Piece WPeon6;
+    private Piece WPeon7;
+    private Piece WPeon8;
+
+    private Piece BRook1;
+    private Piece BHorse1;
+    private Piece BBishop1;
+    private Piece BQueen;
+    private Piece BKing;
+    private Piece BBishop2;
+    private Piece BHorse2;
+    private Piece BRook2;
+    private Piece BPeon1;
+    private Piece BPeon2;
+    private Piece BPeon3;
+    private Piece BPeon4;
+    private Piece BPeon5;
+    private Piece BPeon6;
+    private Piece BPeon7;
+    private Piece BPeon8;
     public Chessboard() {
+        initializeChessboard();
+    }
+    public void initializeChessboard() {
```

Comment:
The names of the pieces are kind of confussiong, as a suggestion maybe you can use "LeftWRook" instead of "WRook1", and similar to the other pieces.